### PR TITLE
added binding ports to rel notes

### PR DESCRIPTION
**Pull request by Amrita42**
- 4.11 only
- [Jira](https://issues.redhat.com/browse/NE-888)
- [Preview](http://file.rdu.redhat.com/~mburke/NE-888new/release_notes/ocp-4-11-release-notes.html)

### CODE REVIEW

File: release_notes/ocp-4-11-release-notes.adoc [hunk 1]
```diff
@@ -99,6 +99,8 @@ By default, the installation program now deploys a Microsoft Azure cluster using
 [id="ocp-4-11-networking"]
 === Networking
 
+This update introduces a new option for Ingress Controllers with `hostnetwork` endpoint strategy where you can host multiple Ingress Controllers on the same worker node using `httpPort`, `httpsPort`, and `statsPort` binding ports.
```
[the remark this answers is not in the document]
Comment (opayne1) [reply release_notes/ocp-4-11-release-notes.adoc:102]: Just a suggestion, but you could break up into 2 sentences for clarity:
```suggestion
This update introduces a new option for Ingress Controllers with `hostnetwork` endpoint strategy. You can now host multiple Ingress Controllers on the same worker node using `httpPort`, `httpsPort`, and `statsPort` binding ports. 
```

**Review by opayne1 — COMMENTED** [review]
Looks good! Just a small question and suggestion. :)

File: release_notes/ocp-4-11-release-notes.adoc (hunk 1 above)
[the remark this answers is not in the document]
Comment (Amrita42) [reply release_notes/ocp-4-11-release-notes.adoc:102]: Good point @opayne1

File: release_notes/ocp-4-11-release-notes.adoc [hunk 2]
```diff
@@ -98,6 +98,9 @@ By default, the installation program now deploys a Microsoft Azure cluster using
 
 [id="ocp-4-11-networking"]
 === Networking
+==== New option for Ingress Controllers with `hostnetwork` endpoint
```
Comment (stevsmit) [inline release_notes/ocp-4-11-release-notes.adoc:101]: We don't use backticks in headings: https://github.com/openshift/openshift-docs/blob/main/contributing_to_docs/doc_guidelines.adoc#quick-markup-reference

**Comment by Amrita42** [comment]
@stevsmit done.

**Pull request merged**